client/rpc/src/lib.rs

```diff
@@ -36,6 +36,118 @@ use rustc_hex::ToHex;
 use pallet_evm::ExitReason;
 use sha3::{Digest, Keccak256};
 
+use sp_runtime::traits::{Block as BlockT, BlakeTwo256, Zero, UniqueSaturatedInto};
+use sp_storage::StorageKey;
+use sp_blockchain::HeaderBackend;
+use sp_api::{BlockId, HeaderT};
+use sc_network::ExHashT;
+use sc_client_api::backend::{StorageProvider, Backend, StateBackend};
+use fc_rpc_core::types::BlockNumber;
+use fp_storage::PALLET_ETHEREUM_SCHEMA;
+
+use jsonrpc_core::Result as RpcResult;
+use codec::Decode;
+use pallet_ethereum::EthereumStorageSchema;
+use std::marker::PhantomData;
+
+pub struct FrontierBackendClient<B: BlockT, C, BE, H: ExHashT> {
```

Comment:
Is there a particular reason we have this struct? Maybe we can move all those functions as static functions?

Reply:
Not in particular, but I'd rather have them under some namespace, either a struct or a module I am ok with both.

Reply:
How about a module? It just feels slightly weird to have a struct but never use it.

Reply:
Yeah sounds good!